.github/workflows/tests.yml

```diff
@@ -0,0 +1,200 @@
+name: Tests
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+
+jobs:
+
+  phpunit:
+
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+         image: postgres:9.6-alpine
+         options: --name=postgres
+         env:
+           POSTGRES_DB: testing
+           POSTGRES_USER: talentcloud
+           POSTGRES_PASSWORD: talentcloud
+         ports:
+          - 5432:5432
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.2
+        extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, bcmath, soap, intl, gd, exif, iconv, imagick
+        tools: composer:v1
+
+    - name: Copy .env
+      run: cp .env.travis .env
+
+    - name: Cache Composer packages
+      id: composer-cache
+      uses: actions/cache@v2
+      with:
+        path: vendor
+        key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-php-
+
+    - name: Install dependencies
+      if: steps.composer-cache.outputs.cache-hit != 'true'
+      uses: MilesChou/composer-action/7.2/install@master
+      with:
+        args: install --no-interaction --prefer-dist --no-progress --no-suggest
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.6.0
```

Comment:
I think [14 is the current LTS version](https://nodejs.org/en/). Can we pin this to LTS so things don't break as we make future updates?